### PR TITLE
dspdfviewer 1.13.1 (new formula)

### PR DESCRIPTION
**Pull request by wwwdata**
This is my new formula to compile this excellent latex-beamer pdf viewer. If anybody has an idea how to write a better test, please let me know. There also is a ticket in the dspdfviewer repo for it https://github.com/dannyedel/dspdfviewer/issues/53

### CODE REVIEW

**Comment by dunn** [comment]
We can sometimes simulate interactivity with `IO.popen`; datomic is one example: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/datomic.rb#L30-L34

File: Library/Formula/dspdfviewer.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  url "https://github.com/dannyedel/dspdfviewer/archive/v1.13.1.tar.gz"
+  sha256 "333588de0316cfdb5821b8484ee55690dfa3c7431b67c126bfdbe9c9cc3e1ed4"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "poppler" => "with-qt"
+
+  def install
+    mkdir "build" do
+      system "cmake", ".."
```
Comment (DomT4) [inline Library/Formula/dspdfviewer.rb]: Please pass `*std_cmake_args` here as well. You're currently trying to install directly to `/usr/local`.

**Comment by wwwdata** [comment]
Hi, I pushed a new commit. However travis is currently still failing. Don't know exactly why. There are some timeouts and github api errors shown up.

**Comment by DomT4** [comment]
You can ignore Travis. Jenkins is the important one still for now, and it passed there.

**Comment by wwwdata** [comment]
ok, so can this be merged?

File: Library/Formula/dspdfviewer.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  desc "Dual-Screen PDF Viewer for latex-beamer"
+  homepage "http://dspdfviewer.danny-edel.de"
+  url "https://github.com/dannyedel/dspdfviewer/archive/v1.13.1.tar.gz"
+  sha256 "333588de0316cfdb5821b8484ee55690dfa3c7431b67c126bfdbe9c9cc3e1ed4"
+
+  head "https://github.com/dannyedel/dspdfviewer.git"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "poppler" => "with-qt"
```
Comment (DomT4) [inline Library/Formula/dspdfviewer.rb]: Can this use `qt5` at all instead of `qt`, out of curiosity?

Comment (wwwdata) [inline Library/Formula/dspdfviewer.rb]: afaik the support is there in the master branch but not in the latest release. So if it's stable we should be able to switch to qt5

**Comment by DomT4** [comment]
It'd be great to get that `qt5` support done in future; Qt aren't supporting Qt4 with anything but security fixes any more and whilst we've added a patch for El Cap it won't stay functional forever sadly.

Merged in https://github.com/Homebrew/homebrew/commit/43943ac922549cc11bffb57ca68ac94fe7be2434. Thank you for your contribution to Homebrew @wwwdata; we appreciate it! :smiley_cat:

**Pull request closed without merging**